ee/tabby-webserver/src/axum/extract.rs

```diff
@@ -19,6 +19,21 @@ where
     type Rejection = Rejection;
 
     async fn from_request_parts(req: &mut Parts, _: &B) -> Result<Self, Self::Rejection> {
+        let access_token = req
+            .uri
+            .query()
+            .and_then(|q| {
+                querystring::querify(q)
+                    .into_iter()
+                    .filter(|(k, _)| *k == "access_token")
```

Comment:
@liangfung can use `/integrations/github/connect?access_token=....` for authorization